app/controllers/appeals_base_controller.rb

```diff
@@ -7,6 +7,13 @@ class AppealsBaseController < ApplicationController
   include ActionController::Serialization
   before_action { authorize :appeals, :access? }
 
+  class << self
+    def exception_hash(exception)
+      %i[message backtrace key response_values original_status original_body]
```

Comment:
Will the 'key' value present in this array clash with the key arg below?

Reply:
shouldn't! unless i'm misunderstanding you.

trying to capture all of these (if they're present)
https://github.com/department-of-veterans-affairs/vets-api/blob/8546477896cc1247aa5d2d54866007df13eb5bec/lib/common/exceptions/backend_service_exception.rb#L12

Reply:
but agreed --it's a funny situation that the variable `key` will get set to the symbol `:key` while reducing that array. I don't think it's funny enough to change my variable name, though. I guess i could go to `k` if our linter allows